src/dbnode/integration/disk_cleanup_multi_ns_test.go

```diff
@@ -133,7 +135,21 @@ func TestDiskCleanupMultipleNamespace(t *testing.T) {
 
 	log.Infof("creating commit log and fileset files")
 	shard := uint32(0)
-	writeCommitLogs(t, filePathPrefix, commitLogTimes)
+	for _, clTime := range commitLogTimes {
+		// Need to generate valid commit log files otherwise cleanup will fail.
+		data := map[xtime.UnixNano]generate.SeriesBlock{
+			xtime.ToUnixNano(clTime): nil,
+		}
+		writeCommitLogDataSpecifiedTS(
+			t,
+			testSetup,
+			testSetup.storageOpts.CommitLogOptions().SetFlushInterval(defaultIntegrationTestFlushInterval),
```

Comment:
super nit: Maybe declare these options before the `for` loop?  Just to avoid this super long line and all:

Reply:
sure